### PR TITLE
Collab request 10 04 2023

### PR DESCRIPTION
**Pull request by vinit-churi**
hi vinit-churi here, I have previously another pull request open but I have close it, please review this new one and let me know if anything missing

### CODE REVIEW

File: Contributors/README.md
Comment (AmrElsayyad) [inline Contributors/README.md:1]: Discard the changes in this file and you're good to go.

**Review by AmrElsayyad — CHANGES_REQUESTED** [review]
Discard changes in `Contributors/README.md` file and you're good to go.

**Comment by InimicalPart** [comment]
Let me know if you need some help!

**Comment by InimicalPart** [comment]
There we go I fixed it
For the future: https://stackoverflow.com/questions/39459467/remove-a-modified-file-from-pull-request

**Review by InimicalPart — APPROVED** [review]
I'll be sending you a collaboration invite shortly 😂

**Pull request merged**